Objective-C/Tests/MultipeerReplicatorTest.m

```diff
@@ -85,13 +85,13 @@ - (void) tearDown {
 
 // TODO: Too flaky, enable after all LiteCore issues have been handled.
 - (BOOL) isExecutionAllowed {
-#if TARGET_OS_SIMULATOR
+#if TARGET_OS_SIMULATOR || TARGET_OS_MACCATALYST || TARGET_OS_OSX
```

Comment:
So this will only work for an iOS, iPadOS or tvOS devices. As we don't support iPadOS and tvOS, maybe we can have this check only as?
```
#if TARGET_OS_IPHONE 
return self.keyChainAccessAllowed
#else
return NO
```